DNN Platform/DotNetNuke.Abstractions/Utilities/IFileSystemUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Utilities
+{
+    using System;
+
+    /// <summary>
+    /// File system utility.
+    /// </summary>
+    public interface IFileSystemUtils
```

Comment:
This creates a new public interface for the platform, is this in alignment with how we want to expose file-system API's?  

Part of me believes for the internal, that this should be an internal element to the insaller?

Reply:
Agreed. I had added this abstraction just for the sake of unit testing, so I could have made it internal.
I have re-implemented this now, and I found some existing internal classes and interfaces in `DNN Platform\Library\Common\Internal`, so I moved this interface there.